lib/index.js

```diff
@@ -160,13 +177,10 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       // if `authenticationMethod` is `token-client-registration` then
       // register token client for it
       if(authenticationMethod === 'token-client-registration') {
-        // generate a client id
-        const clientId = await generateId({fixedLength: true});
         // save it to the database
         await brAuthnToken.clients.set(
           {actor: null, email, account, clientId, authenticated: true});
         // set a cookie in the response
-        const {name, options} = cfg.cookies.clientId;
         res.cookie(name, clientId, options);
```

Comment:
We shouldn't need to set the cookie here, right? We only got to this line because the client already has the cookie set in their browser.

```suggestion
```